src/form/ChestAPI/type/util/InvMenuTypeHelper.php

```
@@ -0,0 +1,52 @@
+	 * @param World $world
+	 * @param Vector3 $position
+	 * @param list<Facing::DOWN|Facing::UP|Facing::NORTH|Facing::SOUTH|Facing::WEST|Facing::EAST> $sides
+	 * @return Generator<Vector3>
+	 */
+	public static function findConnectedBlocks(string $tile_id, World $world, Vector3 $position, array $sides) : Generator{
+		if($tile_id === "Chest"){
+			// setting a single chest at the spot of a pairable chest sends the client a double chest
+			// https://github.com/Muqsit/InvMenu/issues/207
+			foreach($sides as $side){
+				$pos = $position->getSide($side);
+				$tile = $world->getTileAt($pos->x, $pos->y, $pos->z);
```

PHPStan (2.1.15)

Parameter #1 $x of method pocketmine\world\World::getTileAt() expects int, float|int given.


(argument.type)